experiments/integrated/ozark/ozark.jl

```diff
@@ -62,6 +63,53 @@ soil_ps = Soil.EnergyHydrologyParameters{FT}(;
 soil_args = (domain = soil_domain, parameters = soil_ps)
 soil_model_type = Soil.EnergyHydrology{FT}
 
+# Soil microbes model
+soilco2_type = Soil.Biogeochemistry.SoilCO2Model{FT}
+
+soilco2_ps = SoilCO2ModelParameters{FT}(;
+    P_sfc = P_sfc,
+    ν = soil_ν, # same as soil
+    θ_a100 = θ_a100,
+    D_ref = D_ref,
+    b = b,
+    D_liq = D_liq,
+    # DAMM
+    α_sx = α_sx,
+    Ea_sx = Ea_sx,
+    kM_sx = kM_sx,
+    kM_o2 = kM_o2,
+    O2_a = O2_a,
+    D_oa = D_oa,
+    p_sx = p_sx,
+    earth_param_set = earth_param_set,
+);
+
+# soil microbes args
+Csom = (z, t) -> eltype(t)(5.0)
+
+# C = FT(100)
+
+soilco2_top_bc =
+    Soil.Biogeochemistry.SoilCO2StateBC((p, t) -> eltype(t)(0.0004)) # top BC set to atmospheric co2 in mol per mol
```

Comment:
Should this be set to the atmospheric co2 (your comment)?

Reply:
yes - should we grab that from `PrescribedAtmosphere`?

Reply:
I tried `atmos_co2` which is our Spline1D created on co2 data, but it bugs. I'll mark this as resolved once we find what works (probably a type problem)